ros/lib/utils.py

```diff
@@ -80,20 +78,22 @@ def user_data_from_identity(identity):
     return identity['user']
 
 
-def validate_type(value, type_):
+def validate_system(reporter, msg):
```

Comment:
```suggestion
def is_valid_payload(is_ros, cloud_provider):
```

This method needs  just`is_ros` & `cloud_provider` while validation.
Which objects/variables to pass as method arguments, it varies case to case basis and it's judgement call one can decide.

When either reporter's name changed or message object structure changes, one needs to touch this method. In my opinion, this method shouldn't care about getting `is_ros` & `cloud_provider` values . Instead of reporter and message, will it make sense to send only these two values directly.

Reply:
Makes sense!